### PR TITLE
Chore test reliability

### PR DESCRIPTION
**Pull request by AccaliaDeElementia**
Remove the need for mocha `--exit` parameter

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -194,6 +194,9 @@ module.exports = {
 };
 
 if (require.main === module) {
+	// eslint-disable-next-line no-console
+	println = (msg) => console.log(msg);
```
Comment (yamikuronue) [inline src/server.js]: Good solution 👍

Comment (AccaliaDeElementia) [reply src/server.js]: Thanks! :-)

File: src/server.js [hunk 2]
```diff
@@ -166,18 +167,17 @@ function setup(config) {
...
 }
 
 /**
  * Stop the server
  */
 function stop() {
-	server.close();
-	// eslint-disable-next-line no-console
-	console.log('Server stopped');
+	server.close(() => {
```
Comment (Bluenaxela) [inline src/server.js]: This technically seems a bit backwards to me. Doing DB teardown before server close, leaves greater opportunity for there to be http requests that are accepted but error out due to DB teardown. I would think it's better to stop accepting http requests, and then close the DB.

Comment (Bluenaxela) [reply src/server.js]: Wait... I misread the code, it's the right way, hah. Clearly too sleepy.

Comment (AccaliaDeElementia) [reply src/server.js]: @Bluenaxela sleep! i'ts important! 🍊

**Pull request merged**